localstripe/resources.py

```diff
@@ -3144,6 +3144,30 @@ def _calculate_amount_in_tier(self, quantity, index):
         t = self.plan.tiers[index]
         return int(t['unit_amount']) * quantity + int(t['flat_amount'])
 
+    @classmethod
+    def _api_list_all(cls, url, subscription=None, limit=None,
+                      starting_after=None, ending_before=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        # to return 404 if not existant
+        Subscription._api_retrieve(subscription)
```

Comment:
Please move these two lines after the `try/except` like in the other interfaces.